src/GafferOSL/ShadingEngine.cpp

```diff
@@ -912,7 +927,62 @@ OSL::ShadingSystem *shadingSystem()
 
 	g_shadingSystem->attribute( "commonspace", "object" );
 
-	return g_shadingSystem;
+	bool requestBatch = true;
+	if( const char *requestBatchVar = getenv( "GAFFER_OSL_USESIMD" ) )
+	{
+		requestBatch = std::string( requestBatchVar ) != "0";
+	}
+
+#ifdef SHADE_BATCHED
+		// If we wanted to request fused-multiply-add, we would request it here with:
+		// `g_shadingSystem->attribute( "llvm_jit_fma", 1 );`
+		// This is a small improvement in performance ( I couldn't detect a change in my test ),
+		// with the downside that the floating point rounding is slightly different.  The
+		// small difference in rounding is actully more likely to be better than to be worse, so
+		// it all sounds pretty good ... except that the slight difference in rounding may vary
+		// between architectures ... this could be pretty bad for a facility with a heterogenous
+		// farm ... getting a subtle flicker because frames that hit out of date farm blades
+		// render ever-so-slightly darker is not much fun.
+		g_shadingSystem->attribute( "llvm_jit_fma", 1 );
```

Comment:
This seems to be unrelated to the rest of the commit, and the comment implies that we don't want to request FMA, while the code looks like it is requesting it.

Reply:
Oh, good catch.  Actually removed the setting of fma in 8c84600